bitsv/network/services/network.py

```diff
@@ -1,8 +1,13 @@
 import requests
-from .bitindex import BitIndex
 
-DEFAULT_TIMEOUT = 30
+from .bitindex3 import BitIndex3
+
 BSV_TO_SAT_MULTIPLIER = 100000000
+DEFAULT_TIMEOUT = 30
+DEFAULT_APIS = {
+    'BitIndex3': None,
+    #'Whatsonchain': None
+}
```

Comment:
I have pretty mixed feelings about adding services that require API keys. What extra features do you get? Why are API keys necessary for these services?

I als think that if we're going to have this at all, we should be passing dicts and not None/api_key. If we're going to give more flexibility we might as well give a lot more.

So for example:
```
DEFAULT_APIS = {
    'BitIndex3': None,
    #'Whatsonchain': None
}
```

Would be fine. Or:

```
DEFAULT_APIS = {
    'BitIndex3': {'api_key': 'foo',
    #'Whatsonchain': None
}
```
Of course wouldn't have a default API key set. I guess None is fine for a default, but if an API key is passed I think it should be in a dict format to extend options an maybe also have retry logic per service.

Which maybe apis should be an ordered list so we can do ordering... hmm.

Reply:
re: BitIndex API key...

The API key is NOT needed except for the xpub and the webhook API calls.

All the regular utxo/transaction/block/etc calls do not need an API key.

Thanks,

Reply:
I've removed this field in #25 . Thanks. I don't know what I was thinking... I thought it would be a pathway to future proofing and BIP32 support but there's just no easy way to do that via PrivateKey and PrivateKeyTest classes anyway so was a dumb idea to even mess about with it. I do want to have nice BIP32 support added but not like this and may even fall outside the scope of bitsv entirely (may need a separate repo that extends bitsv for example).